The ?reconnect option actually belongs to mysql

Jonty opened the pull request:
This is an artifact of the postgres/mysql support being split and not tested, it will cause postgres connections to fail.

keymon commented:
Can you enable travis on this repository?

keymon commented:
In order to avoid incidents in the future, I think it is better if we "disconnect" this repository from the `cf-platform-eng` one, and we add a reference in the README.

This way we won't do accidental PR to upstream and the search in github will work.

keymon commented:
How to test:

```
go get github.com/alphagov/paas-rds-broker
cd $GOPATH/src/github.com/alphagov/paas-rds-broker
git checkout 117512347_rds_service_broker
go test
go build .
```

keymon commented:
I asked @alext to enable travis

combor commented:
> In order to avoid incidents in the future, I think it is better if we "disconnect" this repository

I'm not sure about this one, what are pros and cons of doing this apart from avoiding creating pull requests for the origin?

keymon commented:
> what are pros and cons of doing this apart from avoiding creating pull requests for the origin?

We won't do any accidental PR to upstream and the search in github will work.

keymon commented:
Will merge this, to carry on reviewing the rest.

keymon commented:
Same than: https://github.com/alphagov/paas-aws-broker-boshrelease/pull/1#issuecomment-215025733

Jonty committed:
Remove mysql option from postgres connect string

This has been present in the URL since the "initial commit" (79918e5),
where there was a single function used for both MySQL and Postgres.
Since then, the code has been refactored so that there is a separate URL
generation function for MySQL and Postgres, however this option has
remained.

This is not an option that's used by Postgresql[1].  Some client
libraries (eg the libpq C library) will emit a warning with this option
present.  Others (eg the Go github.com/lib/pq library) will error.

The result of this is that any client using the VCAP provided
credentials to connect to the postgres database can fail to parse the
connection string and throw and error - previously we have worked around
this by stripping the parameter before using it, but that is not an
acceptable solution for tenants in production.

[1]http://www.postgresql.org/docs/9.5/static/libpq-connect.html#LIBPQ-PARAMKEYWORDS
